packages/react-native-reanimated/android/src/main/worklets/CMakeLists.txt

```diff
@@ -0,0 +1,143 @@
+cmake_minimum_required(VERSION 3.8)
+
+set(LOCAL_PACKAGE_NAME "worklets")
```

Comment:
Since we're here, how about we remove `LOCAL_PACKAGE_NAME` and use `worklets` directly everywhere? This should make the code more readable.